mlflow/cli.py

```diff
@@ -234,8 +234,12 @@ def _validate_static_prefix(ctx, param, value):  # pylint: disable=unused-argume
               help="Additional command line options forwarded to gunicorn processes.")
 @click.option("--waitress-opts", default=None,
               help="Additional command line options for waitress-serve.")
+@click.option("--expose-prometheus", default=False, is_flag=True,
```

Comment:
Is it possible to change this from a flag to a string argument that defines the target directory for prometheus metrics and sets the appropriate  environment variable via the `env` argument to `exec_cmd`? It would also be helpful if the command creates this directory in case it doesn't exist. These changes would make the feature easier to use.

Reply:
Agreed, it is much easier to use this way. I made the modification.